eng/Version.Details.xml

```diff
@@ -150,9 +150,9 @@
       <Uri>https://github.com/dotnet/emsdk</Uri>
       <Sha>9838ec0843442f761488cfec9cf34612c9f675e6</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build" Version="0.1.0-alpha.1.21460.1" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build" Version="0.1.0-alpha.1.21318.1" CoherentParentDependency="Microsoft.NET.Sdk">
```

Comment:
@sfoslund Is this downgrade expected?

Reply:
It looks like it's coming from source build, @MichaelSimons is this expected?

Reply:
No.  It looks like I caused this by these changes - https://github.com/dotnet/installer/pull/11965/files.  I think I need to backport these into the SDK to correct this.  I don't see a darc subscription for this flow into the SDK, it is a manual update.